pages/pipelines/create_your_own.md

```diff
@@ -61,27 +61,25 @@ You'll create a new pipeline that uploads the pipeline definition from your repo
 
 To create a new pipeline:
 
-1. Navigate to the [Buildkite dashboard](https://buildkite.com/) by selecting _Pipelines_.
-1. Select <%= image "new-pipeline-button.svg", alt: "The build page" %>
- _New Pipeline_.
+1. Select _Pipelines_ to navigate to the [Buildkite dashboard](https://buildkite.com/).
+1. Select _New pipeline_.
 
-    If you're prompted to connect your repositories, we recommend completing that first, but you can always connect them later from the settings.
+    **Note:** On the _New Pipeline_ page, if you're prompted to connect your repositories, it is recommended you do that first. You can always connect them later from your pipeline's settings.
     After connecting your repositories, you can select them from the dropdown during pipeline creation and enable automatic webhook creation.
 
-1. Enter the details you want for the pipeline. You can always change these later in the pipeline settings.
-1. Select _Create Pipeline_.
-1. In the steps editor, ensure there's a step to upload the definition from your repository:
+1. Enter your pipeline's details in the respective _Name_ and _Description_ fields. You can always change these details later from your pipeline's settings.
```

Comment:
I wonder if we should mention selecting their repository here. That's now the top field on this screen.

Reply:
Good point, as per the inline **Note** above, do you need to have 'connected your repositories' first at this point, or could you paste any repo URL into this field? I suspect the latter...

Reply:
Yeah, you're right. You could paste in any URL. The benefit of connecting your repositories is that you'll get a dropdown list with options to choose from.

Reply:
I ended up re-writing this note a tad to clarify that it's actually a Git repo account you're connecting to (implied by the other dropdown option being **Any account**).

I then included an additional step about specifying the repo to build from.